build/build.sh

```diff
@@ -39,7 +49,25 @@ if [[ -n "$TRAVIS_TAG" ]] && [[ $TRAVIS_TAG != *"RC"* ]]; then
 fi
 
 # Get the version details
-VERSION="$(cat $GOPATH/src/github.com/openebs/cstor-operators/VERSION)"
+#VERSION="$(cat $GOPATH/src/github.com/openebs/cstor-operators/VERSION)"
+
+## Populate the version based on release tag
+## If travis tag is set then assign it as VERSION and
+## if travis tag is empty then mark version as ci
+if [ -n "$TRAVIS_TAG" ]; then
+    # Trim the `v` from the TRAVIS_TAG if it exists
+    # Example: v1.10.0 maps to 1.10.0
+    # Example: 1.10.0 maps to 1.10.0
+    # Example: v1.10.0-custom maps to 1.10.0-custom
+    VERSION="${TRAVIS_TAG#v}"
+else
+    VERSION="dev"
+fi
+
+echo "Building for VERSION ${VERSION}"
+## Below line will help to get current version for various binaries
+echo "${VERSION}" > "$VERSION_FILE_PATH"
```

Comment:
why do we need version file now? Just setting the VERSION variable should be sufficient.

Reply:
In code, we are making use of the VERSION file because of this reason generated only during build time and deleted after that.

Reply:
can you show me where we are using?

Reply:
![image](https://user-images.githubusercontent.com/13776912/85567889-daa25f00-b64e-11ea-8e07-dd21c62bac52.png)

Reply:
> why do we need version file now? Just setting the VERSION variable should be sufficient.

Correct!! setting the VERSION variable is good enough!!